package.json

```diff
@@ -1,8 +1,40 @@
 {
   "name": "bats",
-  "version": "v1.0.1",
+  "version": "0.4.0",
```

Comment:
Any reason why this is set back from `v1.0.1`?

Reply:
I'm going to publish both 1.0.0 and 1.0.1 versions once the rest of the metadata is approved, so (for this PR which _won't_ be merged) I'm keeping it at the old version. That way if I forget to bump it before attempting to publish, it'll error (existing version).

Reply:
(FYI, I intend to publish from the commits actually tagged per version)